### PR TITLE
Add docstrings, reinstate poll logic (!)

### PR DESCRIPTION
**Pull request by bmcorser**
Assigning @Singletoned for the docstrings and @canni for the poll logic (I spotted it the second time round!)

### CODE REVIEW

**Comment by uktitools** [comment]
Can one of the admins verify this patch?

**Comment by bmcorser** [comment]
Sorry @adammalinowski @AaronWitter buttterfingers

File: korben/bau/poll.py [hunk 1]
```diff
@@ -65,19 +68,31 @@ def poll_entities(self, entities=None):
...
+        Main polling logic:
+
+        Download pages of PAGE_SIZE entries by increasing offset until less
+        than PAGE_SIZE combined updates and inserts happen, which is taken as
+        indication that that entity doesn’t have any more changes.
+        '''
+        updates = offset = 0
+
+        while not updates < self.PAGE_SIZE or offset == 0:
+            updates =\
```
Comment (Singletoned) [inline korben/bau/poll.py]: Don't escape line-breaks, use brackets and implicit line continuation instead.

File: korben/bau/poll.py [hunk 2]
```diff
@@ -90,13 +105,15 @@ def reverse_scrape_page(self, table, col_names, primary_key, offset):
...
                 LOGGER.debug('Row in %s doesn’t exist', table.name)
-                result = self.conn.execute(table.insert().values(**row))
+                result = self.db.execute(table.insert().values(**row))
                 assert result.rowcount == 1
                 leeloo.send(  # to django
-                    *etl.main.from_odata(table, [row[primary_key]], dont_load=True)
+                    *etl.main.from_odata(
+                        table, [row[primary_key]], dont_load=True
+                    )
                 )
```
Comment (Singletoned) [inline korben/bau/poll.py]: It doesn't seem necessary for this to take 5 lines.

**Pull request merged**